### ISSUE TITLE
Can't deply bower webjar tinymce-dist  version 4.3.1

### ISSUE THREAD

**Issue by nikomiranda** [webjars/webjars#1265]
When deploying the following message 

       Failed!
      Received fatal alert: handshake_failure to https://www.tinymce.com/license
      If you feel you have reached this failure in error, please file an issue: https://github.com/webjars/webjars/issues
            
      Determined Artifact Name: tinymce-dist
      Starting Deploy'

Until version 4.3.0 everything was okey.

**Comment by jamesward** [comment]
[The license metadata](https://github.com/tinymce/tinymce-dist/blob/master/bower.json#L4) points to a URL that 404's.  I'm not sure what this has to do with a `handshake_failure` but I'm betting this is the issue.  This will have to be fixed upstream.

**Comment by nikomiranda** [comment]
Now the license metadata is doing a redirect to another site https://raw.githubusercontent.com/tinymce/tinymce/master/LICENSE.TXT and still doesn't work.

**Comment by jamesward** [comment]
It seems that Java and OpenSSL are not compatible with the certificate they are using.  Here is what I get from openssl:
```
~ $ openssl s_client -connect www.tinymce.com:443 -showcerts
CONNECTED(00000003)
13356:error:14077410:SSL routines:SSL23_GET_SERVER_HELLO:sslv3 alert handshake failure:/SourceCache/OpenSSL098/OpenSSL098-52.40.1/src/ssl/s23_clnt.c:593:
```

I'm not sure what we can do about this.  Can you file an issue with them?

**Comment by nikomiranda** [comment]
Seems to be fixed now (deployed right now 4.3.3) thanks!
